config/rubocop.yml

```diff
@@ -69,6 +69,9 @@ SignalException:
 BlockDelimiters:
   Enabled: false
 
+Metrics/BlockLength:
+  Enabled: false
+
```

Comment:
These changes to rubocop.yml and reek.yml don't seem necessary to make the tests pass.